docs/data/base/components/trap-focus/trap-focus.md

```diff
@@ -84,3 +84,9 @@ When auto focus is disabled—as in the demo below—the component only traps th
 The following demo uses the [`Portal`](/base/react-portal/) component to render a subset of the `TrapFocus` children into a new "subtree" outside of the current DOM hierarchy, so they are no longer part of the focus loop:
 
 {{"demo": "PortalTrapFocus.js"}}
+
+## Using a toggle inside the trap
+
+The most common use case for the `TrapFocus` component is to maintain focus within a [`Modal`](/base/react-modal/) component which is entire separete from the element which would open the modal. However, there are use cases where you may want to have a toggle button for the `open` prop of the `TrapFocus` component which is stored inside that component.
```

Comment:
```suggestion
The most common use case for the `TrapFocus` component is to maintain focus within a [modal](/base/react-modal/) component that is entirely separate from the element that opens the modal. 
But you can also create a toggle button for the `open` prop of the `TrapFocus` component that is stored inside of the component itself, as shown in the following demo:
```

Reply:
Done, except for one thing. I think it should be "[`Modal`](/base/react-modal/)" and not "[modal](/base/react-modal/)" because directly above this section, the [Portal component is referenced](https://github.com/mui/material-ui/blob/master/docs/data/base/components/trap-focus/trap-focus.md?plain=1#L84) in this exact same way. 😄